datasets/autshumato_parallel/README.md

```diff
@@ -0,0 +1,22 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- expert-generated
+languages:
+- en
+- tn
+- ts
+licenses:
+- cc-by-2.5
+multilinguality:
+- multilingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+task_ids:
+- machine-translation
+---
```

Comment:
could you add the paragraph titles of the dataset card ?
you can leave the content with `[More Information Needed]`